ProcessLib/PhaseField/PhaseFieldProcess-impl.h

```diff
@@ -156,21 +295,44 @@ void PhaseFieldProcess<DisplacementDim>::assembleWithJacobianConcreteProcess(
     const double dxdot_dx, const double dx_dx, GlobalMatrix& M, GlobalMatrix& K,
     GlobalVector& b, GlobalMatrix& Jac)
 {
-    // DBUG("AssembleJacobian PhaseFieldProcess.");
-
     std::vector<std::reference_wrapper<NumLib::LocalToGlobalIndexMap>>
-       dof_tables = {std::ref(*_local_to_global_index_map)};
+        dof_tables;
+
+    // For the monolithic scheme
+    if (_use_monolithic_scheme)
+    {
+        DBUG("AssembleJacobian PhaseFieldProcess for the monolithic scheme.");
+        dof_tables.emplace_back(*_local_to_global_index_map);
+    }
```

Comment:
This if-else-block for dof-table creation seems to be identical to the one in assembleConcreteProcess(). Could this duplication be avoided?